impeller/entity/contents/atlas_contents.cc

```diff
@@ -94,8 +94,7 @@ bool AtlasContents::Render(const ContentContext& renderer,
     using FS = PorterDuffBlendPipeline::FragmentShader;
 
 #ifdef IMPELLER_DEBUG
-    pass.SetCommandLabel(
-        SPrintF("DrawAtlas Blend (%s)", BlendModeToString(blend_mode)));
```

Comment:
I don't care about the distinction in the label because I can see the blend mode via the shader parameters or draw call, but you're right that we could be a bit smarter here and compute a better label if we needed to.

Reply:
Yeah, this was just a nit. If we already have this info elsewhere, then nothing needed.